config-db.yml

```diff
@@ -45,6 +45,15 @@
         - lookup('env', 'PRIMARY_IP_ADDR') is not defined or lookup('env', 'PRIMARY_IP_ADDR') | length == 0
       tags: primary-db
 
+    - include_role:
+        name: workload-agent
+      when:
+        - ansible_facts['product_name'] == "Google Compute Engine"
+        - create_db | bool
```

Comment:
We can install the agent even if not creating a database in the toolkit;  it can detect databases migrated in the future.  We can't set up metric collection though, naturally.